FogCreek/Glitch-Community
# Fine paradox: Disable Collection Sorting on < mediumSmallViewport

## Links
* Remix link: https://fine-paradox.glitch.me/

## GIF/Screenshots:
![scaling](https://user-images.githubusercontent.com/519336/68507397-ce8d9c80-0239-11ea-907a-3548f8f743e1.gif)

## Changes:
* Disable sorting of projects in collections on small screens.  There's no reason to sort on small screens anyway, since the single-column layout makes it difficult to reorder projects, and it leads to accidental sorting when users are browsing collections on mobile.

## How To Test:
* Log into the remix, open one of your collections, and ensure you can't sort when your screen is smaller than 592px.
